ci/Dockerfile.ledger

```diff
@@ -1,9 +0,0 @@
-# Taken from bitcoindevkit/rust-hwi
```

Comment:
Can we avoid removing this dockerfile and speculos' `automation.json`? I'm doing some changes to hwi in #1161, so I will need them again

Reply:
Done